src/lib/pages/backoffice/Acquisition/Order/OrderEditor/OrderForm/OrderForm.js

```diff
@@ -114,7 +114,7 @@ export class OrderForm extends Component {
 
   render() {
     const { successSubmitMessage, data, title, pid, isCreate } = this.props;
-
+    data.metadata.order_lines = data.metadata.resolved_order_lines;
```

Comment:
Why do we re-assign a renamed value in `data.metadata` would something like the following work?

```suggestion
    const { resolved_order_lines: order_lines } = data.metadata;
```

Reply:
I think he is assigning actually here, not renaming (replacing existing `order_lines` with `resolved_order_lines` to display resolved patron and document in the form)
which I think it is a bit confusing in general, we are discussing on chat